README.md

```diff
@@ -4,3 +4,67 @@
 
 [`axum`]: https://github.com/tokio-rs/axum#readme
 [jsonwebtoken]: https://github.com/Keats/jsonwebtoken#readme
+
+## Usage
+
+1. Install `axum-jsonwebtoken`:
+
+  ```sh
```

Comment:
**nitpick:** If you double indent the code block (i.e. 4 spaces instead of 2) it will align it nicer.